### PR TITLE
Compose DownloadCard

### PR DESCRIPTION
**Pull request by VDostoyevskiy**

### CODE REVIEW

**Review by FooIbar — COMMENTED** [review]
Drag and drop does not work.
Just disable dragging since we'll migrate to compose anyway?

File: app/src/main/kotlin/com/hippo/ehviewer/ui/main/DownloadCard.kt [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+                    ProvideTextStyle(MaterialTheme.typography.labelMedium) {
+                        if (total <= 0 || finished < 0) {
+                            LinearProgressIndicator(
+                                modifier = Modifier.constrainAs(progressBarRef) {
+                                    bottom.linkTo(actionsRef.top)
+                                },
+                            )
+                        } else {
+                            LinearProgressIndicator(
+                                progress = { total.toFloat() / finished },
```
Comment (FooIbar) [inline app/src/main/kotlin/com/hippo/ehviewer/ui/main/DownloadCard.kt:219]: Typo?

Comment (VDostoyevskiy) [reply app/src/main/kotlin/com/hippo/ehviewer/ui/main/DownloadCard.kt]: LMAO, confusing why it does not work for a while

**Review by FooIbar — APPROVED** [review]
LGTM, except I think the `DragHandle` is kinda ugly and prefer `Reorder`.

**Pull request merged**